config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -481,6 +481,50 @@ EOF
           privileged: true
 $(resources_for_kubernetes master   "        ")
 EOF
+
+    if [[ $repo != "csi-driver-host-path" ]]; then
+      cat >>"$base/$repo/$repo-config.yaml" <<EOF
+
+  - name: $(job_name "pull" "$repo" "canary")
+    optional: true
+    decorate: true
+    skip_report: true
```

Comment:
This all looks reasonable, except for this: does `skip_report` mean that Prow is not reporting back to GitHub? I think we want the job results to be shown there.

Reply:
Makes sense, done!